hang_man/main.js

```diff
@@ -0,0 +1,114 @@
+const mystery_words_array = [
+  "Malamute",
+  "Poodle",
+  "Labrador",
+  "Spaniel",
+  "Bloodhound",
+  "Sheppard",
+  "Weiner",
+  "Bulldog",
+  "Boxer",
+  "Chihuahua",
+  "Mutt",
+  "Weimaraner",
+  "Pug"
+]
+const mystery_word = mystery_words_array[Math.floor(Math.random()*mystery_words_array.length)].toLowerCase();
+console.log(mystery_word)
+const mystery_letter_array = mystery_word.split("");
+let wrong_attempt_counter = 0;
+let win = 0;
+const you_win_sound = new Audio(`sounds/win.wav`);
+const you_loose_sound = new Audio(`sounds/fart.wav`);
+
+$(document).ready(function (){
+
+  drawKeyboard()
+  drawMysteryWord(mystery_word)
+
+  document.addEventListener('keydown', event => {
+    const { keyCode } = event;
+    guessLetter(keyCode)
+  })
+  document.querySelectorAll('.letter').forEach(node => {
+    node.addEventListener('click', event => {
+      let keyCode = event.currentTarget.id
+      guessLetter(keyCode)
+    }) 
+
+  })
+  document.querySelector("#hint-button").addEventListener("mouseenter", () => {
+    document.querySelector("#hint-button").classList.add("d-none");
+    document.querySelector("#cheater").classList.remove("d-none");
+    document.querySelector("#hint").classList.remove("d-none");
+  });
+  document.querySelector("#cheater").addEventListener("mouseleave", () => {
+    document.querySelector("#hint-button").classList.remove("d-none");
+    document.querySelector("#cheater").classList.add("d-none");
+    document.querySelector("#hint").classList.add("d-none");
+  });
```

Comment:
Are you deliberately avoiding using jQuery? Or are you just not familiar with it yet? I do know some people are now starting to disfavour its use, since many of the features it once offered are now to a certain degree "irrelevant" on modern browsers. However, if your app is meant to work on older browsers, jQuery will save you a lot of grief. I personally still like it quite a bit myself. E.g. the above would become:
```javascript
$('#cheater').on('mouseleave', () => {
  $('#hint-button').removeClass('d-none');
  $('#cheater, #hint').addClass('d-none');
});
```